README.md

```diff
@@ -1 +1,199 @@
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[HTML basic webpage]**
+- **[CSS styling]**
+- **[README.md updating]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+node.js needed
+run npm install
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+
+```sh
+  cd my-folder
+  git clone git@github.com:suthish777/Personal-porfolio.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+
+
+```sh
+  cd my-project
+  npm install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Suthish Prasanna**
+
+- GitHub: [@githubhandle](https://github.com/suthish777)
+- Twitter: [@twitterhandle](https://twitter.com/SuthishPrasanna)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/suthish-prasanna-5b6b2417b)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Improving webpage designe]**
+- [ ] **[updating style.css]**
+- [ ] **[updating pull request]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project... please add star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank... Microverse for this initiative.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+- **[what to do when style.css did'nt work]**
+
+  - [check the spaces and run npm comand for css.]
+
+- **[what to do when I get confused]**
+
+  - [Be calm and search the solutins on the internet]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
[OPTIONAL] Well done on the README 📖 but make sure to fix your MIT License Link as it is broken for now ❌ This is what I get when I try to access it 👇🏽 

![Screenshot 2023-05-03 at 07 38 26](https://user-images.githubusercontent.com/45739963/235905702-ec29b43f-caed-4d33-a16c-8020fa5f033e.png)

This is only OPTIONAL as it was not mentioned previously, but very important to make your project look more professional. 🕴🏽